quickstart/cpp/queryobjects/PortfolioPdx.hpp

```diff
@@ -90,7 +90,9 @@ class TESTOBJECT_EXPORT PortfolioPdx : public PdxSerializable {
 
   int8_t* getArrayZeroSize() { return arrayZeroSize; }
 
-  static PdxSerializable* createDeserializable() { return new PortfolioPdx(); }
+  static std::shared_ptr<PdxSerializable> createDeserializable() {
+    return new PortfolioPdx();
```

Comment:
Hmm.. `std::make_shared`?

Reply:
Good eye, must not be compiled as part of the build because that is definitely a compiler failure.